### PR TITLE
Add eslint config for eslint

### PR DESCRIPTION
**Pull request by quesabe**

### CODE REVIEW

**Comment by quesabe** [comment]
@gvidon The PR includes both the config and its integration into the `ofmt` script. You can see that CI fails, which is caused by the config not being published yet. There remains a chance that something goes wrong and it won't work as expected.

I see two options here:
1. To make it shoother we can separately publish an update with the config only and then when it is available in `npm` I'd test the config and submit the updated script.
2. Publish all at once and then do a fix if needed.

Please let me know which route you prefer. PS: for me the first one seems better.

File: eslint-config-ofmt/eslint.layout.cjs [hunk 1]
```diff
@@ -0,0 +1,24 @@
+/**
```
Comment (gvidon) [inline eslint-config-ofmt/eslint.layout.cjs]: @quesabe I would suggest renaming the file to `eslint.formatting.cjs`, to be extremely explicit about the purpose of the file — it should never include static analysis rules. Also, please add a comment righting the file about the purpose of the file.

**Review by gvidon — CHANGES_REQUESTED** [review]
@Serator can you please test the functionality in a real project?

**Review by gvidon — APPROVED** [review]
@quesabe let's do the second option, seems easier to me. Feel free to merge the PR.

**Pull request merged**